--- golang-1.6/src/crypto/x509/root_linux.go~	2016-02-17 22:35:21.000000000 +0200
+++ golang-1.6/src/crypto/x509/root_linux.go	2016-04-05 13:04:44.231441583 +0300
@@ -11,6 +11,7 @@
 
 // Possible directories with certificate files; all will be read.
 var certDirectories = []string{
+	"/etc/openssl/certs",           // PLD Linux
 	"/etc/ssl/certs",               // SLES10/SLES11, https://golang.org/issue/12139
 	"/etc/pki/tls/certs",           // Fedora/RHEL
 	"/system/etc/security/cacerts", // Android
