--- golang-1.2.1/src/pkg/crypto/x509/root_unix.go~	2014-03-03 02:57:17.000000000 +0200
+++ golang-1.2.1/src/pkg/crypto/x509/root_unix.go	2014-03-13 10:20:37.213363038 +0200
@@ -10,6 +10,7 @@
 
 // Possible certificate files; stop after finding one.
 var certFiles = []string{
+	"/etc/certs/ca-certificates.crt",         // PLD Linux
 	"/etc/ssl/certs/ca-certificates.crt",     // Debian/Ubuntu/Gentoo etc.
 	"/etc/pki/tls/certs/ca-bundle.crt",       // Fedora/RHEL
 	"/etc/ssl/ca-bundle.pem",                 // OpenSUSE
